frontend/src/components/graduations/GraduationLinkCard.tsx

```diff
@@ -0,0 +1,51 @@
+'use client';
+
+import { toDojoDateString, toDojoTimeString } from '@/calendar/displayDate';
+import { Graduation } from '@/database/graduation';
+import CohortIcon from '@/scoreboard/CohortIcon';
+import { Card, CardActionArea, CardContent, Stack, Typography } from '@mui/material';
+import Grid2 from '@mui/material/Unstable_Grid2';
+import Link from 'next/link';
+
+interface GraduationLinkCardProps {
+    graduation: Graduation;
+    to: string;
+}
+
+export const GraduationLinkCard = ({ graduation, to }: GraduationLinkCardProps) => {
+    const { newCohort, displayName, createdAt: graduatedAt } = graduation;
+
+    const dateStr = toDojoDateString(new Date(graduatedAt), undefined);
+    const timeStr = toDojoTimeString(new Date(graduatedAt), undefined, undefined);
```

Comment:
These dates/times should be passing in the viewer's timezone/timeformat

Reply:
This is not available while maintaining that it's publicly accessible pure SSR. I'd really like to maintain that design. Also, this isn't an exposed feature yet and is in there for debugging and future potential. I'd rather not fiddle with it more, but I would also like to keep it in.

Reply:
It has a 'use client' directive so isn't SSR right now anyway. But you can leave it as is for now